packages/useModal/src/useModalHook.tsx

```diff
@@ -5,8 +5,9 @@
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
-import { useContext, useEffect, useState, useCallback, useMemo } from 'react'
-import { ModalContext, ModalType } from './ModalContext'
+import React, { useCallback, useEffect, useMemo, useRef } from 'react'
+import { unmountComponentAtNode } from 'react-dom'
+import ReactDOM from 'react-dom'
```

Comment:
You can combine there two
```suggestion
import ReactDOM, { unmountComponentAtNode } from 'react-dom'
```